lib/node_modules/@stdlib/utils/async/parallel/README.md

```diff
@@ -0,0 +1,297 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Parallel
+
+> Execute functions in parallel, passing the results of all functions to a final callback.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var parallel = require( '@stdlib/utils/async/parallel' );
+```
+
+#### parallel( functionArray, \[options,] done )
+
+Executes functions inside `functionArray` in parallel, passing the results of all functions to a final callback.
+
+The first argument to each function is a `resolve` callback. The `resolve` callback should be invoked upon function completion. The first argument is reserved as an `error` argument (which can be `null` or void). If there is a result, it should be provided as the second argument to the `resolve` callback. Failure to invoke the `resolve` callback will prevent the `done` callback from being invoked.
```

Comment:
This should be moved to the notes section. There is some duplication here with what is below.